terraform/modules/ignition/ignition.tf

```diff
@@ -32,10 +32,15 @@ data "ignition_config" "main" {
   ]
 
   users = ["${data.ignition_user.core.id}"]
+
+  append {
+    source       = "${lookup(var.ignition_extra_config, "source")}"
+    verification = "${lookup(var.ignition_extra_config, "verification", "")}"
+  }
```

Comment:
I assume that the `ignition_config` resource still behaves properly if those are empty?

Reply:
[Filed a bug report](https://github.com/terraform-providers/terraform-provider-ignition/issues/48) against the provider, and using a hacky solution for now (use a empty config).

Reply:
Awesome, thanks! Happy to merge once the other comment is addressed as well.